### DISCUSSION TITLE
[exporter/loki] Is there a way to change de exporter label?

### DISCUSSION THREAD

**Discussion by coolapso** [open-telemetry/opentelemetry-collector-contrib#31799] [Q&A]
Currently loki exporter sets the exporter label to "OTLP" is there a way to change this label value?

**Comment by Frapschen** [comment]
Can you give more detail about your question?

**Reply by coolapso** [reply]
There's a default label called "exporter" which seems to be hard-coded to "OTLP", or at least I couldn't find how to change it without having to use the attributes processor.
